src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java

```diff
@@ -84,25 +84,31 @@ private ConnectionDefinition createConnectionDefinition() {
         final JsonObjectBuilder configJson = getConnectionConfig();
         return createConnectionDefinition(configJson);
     }
+
     private Optional<String> getHostOverride() {
-        return this.absTestSetup.getHostOverride().map(address -> this.exasolTestSetup
-                .makeTcpServiceAccessibleFromDatabase(ServiceAddress.parse(address)).toString());
+        return this.absTestSetup.getHostOverride().map(address -> {
```

Comment:
If I had a wish, then I'd propose to try to remove the use of `Optional` in `AbsTestSetup.getHostOverride()` and `IntegrationTestSetup.getHostOverride()`. But I do not want to block your release and leave it up to you, as this change can also be done in a separate PR at a later point in time.

Reply:
Let's keep that for a later time :) I want to get this out of the door today. Save your wish :)

Reply:
👍

Reply:
https://github.com/exasol/azure-blob-storage-document-files-virtual-schema/issues/20